tide/core/log.cpp

```diff
@@ -58,34 +58,37 @@ extern "C" {
 }
 #endif
 
+#define MAX_LOG_LENGTH 1024
```

Comment:
this used to be a const size_t MAX_LOG_LENGTH = 1024; so don't change it to a \#define